AirCasting/APICommunicator/StreamDownloader.swift

```diff
@@ -7,7 +7,7 @@ import Resolver
 struct StreamWithMeasurements: Decodable {
     let title: String
     let username: String
-    let measurements: [StreamWithMeasurements.Measurements]
+    var measurements: [StreamWithMeasurements.Measurements]
```

Comment:
Why is this changed?

Reply:
![Zrzut ekranu 2022-06-13 o 09 30 03](https://user-images.githubusercontent.com/44276070/173302134-fcfa46b5-6d3b-4804-a8a1-25bd284282d6.png)
Other way I am not able to assign those filter measurements here (CompleteScreenViewModel)